examples/functions/python3/mmlogic-simple/api/protobuf_spec/messages_pb2.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
Why are we adding utf-8 encoding format to these files? I think the default ascii encoding should work for all generated protobuf files, otherwise these generated files won't work.

Reply:
I only changed the Makefile. The rest of the files are updated via `make presubmit`. I'll try to add automation for this a bit later btw.